src/modules/dashboard/components/Members/filterMembers.ts

```diff
@@ -0,0 +1,16 @@
+export const filterMembers = (data, filterValue) => {
```

Comment:
This util function could benefit from some type definition :)

Reply:
Agreed